lib/events/experiences.js

```diff
@@ -2,24 +2,30 @@
 //
 // User events on the experiences widget.
 
-import validate from '../validate';
+import validate from '../validate'
 
 export default class Experiences {
   constructor(tracker) {
-    this.tracker = tracker;
+    this.tracker = tracker
   }
 
   dockToggled(opts) {
-    opts.fullDockToggledParams = JSON.stringify(opts);
+    // FIXME: should be like commented code below
```

Comment:
Why you added FIXME and did not fix the code?

Reply:
The first time i've fixed it - test was failed, and I returned it back but with FIXME.
As you can see in this func we mutate arguments, but it's a bad practice and i need to more investigate how it works before fix it. I've decided to do it but in another task. I think it's a correct. Do you agree with me?